common/supplemental/likelySubtags.xml

```diff
@@ -478,6 +478,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Chipewyan; ?; ? } => { Chipewyan; Latin; Canada }-->
 		<likelySubtag from="chr" to="chr_Cher_US"/>
 		<!--{ Cherokee; ?; ? } => { Cherokee; Cherokee; United States }-->
+		<likelySubtag from="cic" to="cic_Latn_US"/>
+		<!--{ Chickasaw; ?; ? } => { Chickasaw; Latin; United States }-->
```

Comment:
We probably need a ticket to add Chicasaw to the territory-language info for US, if we don't already have one (otherwise this will get overwritten)